### PR TITLE
Added test conduct registration model

### PR DESCRIPTION
**Pull request by patelhr**
**Fixed Issues**

#21161

**Files Changed**

TestConductServiceResponse.cs-Added response class for handling response of server side API.
TrappistDbContext.cs-Added TestAttendees tabel.
test.cs-Added relationship between Test and TestAttendees model class.
TestAttendees.cs-Added model class for test attendees.

**Checks**

- [x] Naming Conventions
- [ ] Server side code comments (proper English, grammar and no spelling mistakes)
- [ ] Client side code comments (proper English, grammar and no spelling mistakes)
- [x] Unused variables, methods, blank spaces and name spaces. 
- [x] Optimal Code and code formatting
- [x] Commit History is proper, no merge is done. 
- [x] Commit/pull request messages should be proper to justify your feature
- [ ] All test cases are executed provided by tester.

**Known Issues**
N/A

**Comments**
N/A

### CODE REVIEW

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [TrappistDbContext.cs#L92](https://github.com/Promact/trappist/blob/672ee789918c9b3ed1e2169e6b8e7ca0d42af4d6/Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs#L92): Use the overloading mechanism instead of the optional parameters. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS2360)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/TestConduct/TestConductServiceResponse.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Promact.Trappist.DomainModel.ApplicationClasses.TestConduct
+{
+    class TestConductServiceResponse
+    {
+        public bool IsSucceed { get; set; }
+        public string ErrorMessage { get; set; }
+        public string LinkID { get; set; }
```
Comment (iroshni) [inline Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/TestConduct/TestConductServiceResponse.cs]: LinkId should be the name

File: Trappist/src/Promact.Trappist.DomainModel/Models/TestConduct/TestAttendees.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        [StringLength(255)]
+        public string LastName { get; set; }
+
+        [StringLength(15)]
+        public string ContactNumber { get; set; }
+
+        [Required]
+        public string RollNumber { get; set; }
+
+        public int TestID { get; set; }
```
Comment (iroshni) [inline Trappist/src/Promact.Trappist.DomainModel/Models/TestConduct/TestAttendees.cs]: TestId should be the name

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [TrappistDbContext.cs#L92](https://github.com/Promact/trappist/blob/d31ed9b9b6fab55addcdc38fad2db7efab2acc1b/Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs#L92): Use the overloading mechanism instead of the optional parameters. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS2360)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [TrappistDbContext.cs#L92](https://github.com/Promact/trappist/blob/279f9755e9ca36779366c2aef805148da3f134d9/Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs#L92): Use the overloading mechanism instead of the optional parameters. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS2360)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Pull request merged**